ui/app/styles/pages/auth-page.scss

```diff
@@ -31,8 +31,10 @@
 
     .icon.brand {
       display: flex;
+      width: scale.$lg-14;
```

Comment:
@heatherlarsen for knowledge, one case in which the `24px` icon is scaled up to `96px`

Reply:
I ran into an issue getting this to run locally, but I'm assuming this is either for the Waypoint logo or the HashiCorp logo. I'd advocate for using a logo in these instances instead of an icon.

Reply:
> I ran into an issue getting this to run locally

@heatherlarsen I’d like to hear more about this. I’ll reach out to you on Slack :)